src/Converter/GhostscriptConverter.php

```diff
@@ -44,10 +44,7 @@ public function __construct(GhostscriptConverterCommand $command, Filesystem $fs
     {
         $this->command = $command;
         $this->fs = $fs;
-
-        if ($tmp) {
-            $this->tmp = $tmp;
-        }
+        $this->tmp = empty($tmp)?sys_get_temp_dir():$tmp;
```

Comment:
Our dependencies (from Symfony) requires PHP 5.3. Also, PHP 5.2 is too old. Are you using it?

What you thing about: `$this->tmp = empty($tmp) ? sys_get_temp_dir() : $tmp;`?